### DISCUSSION TITLE
veLoading no work    "TypeError: this.$veLoading is not a function"

### DISCUSSION THREAD

**Discussion by coreqnq** [Happy-Coding-Clans/vue-easytable#490] [General]
import { VeTable, VePagination, veLoading } from 'vue-easytable'

export default {
  name: 'TablesBasic',
  components: {
    VeTable,
    VePagination,
  veLoading
  },
  data () {
    return {
      loadingInstance: {},
.....
.......
  mounted () {
    core.index()
    this.loadingInstance = this.$veLoading({
      target: document.querySelector('#tablebusiness'),
      name: 'wave'
    })
  },

**Comment by huangshuwei** [comment]
If you want global usage，You need set the veLoading component to the prototype of Vue.
```
import Vue from "vue";
import { veLoading } from "vue-easytable";

Vue.prototype.$veLoading = veLoading;
```

Refer to document:https://happy-coding-clans.github.io/vue-easytable/#/en/doc/base/loading?anchor=global-usage

**Reply by Fox237** [reply]
import Vue from "vue";
import { VeLoading } from "vue-easytable";

Vue.prototype.$veLoading = VeLoading;
似乎V是大写
